docs/VnVPlan/VnVPlan.tex

```diff
@@ -549,8 +549,317 @@ \subsection{Tests for Nonfunctional Requirements}
 be involved? Be specific.}
 
 \subsubsection{Look and Feel Requirements}
+These tests are to ensure that the way the front end is designed adheres to the
+defined requirements with respect to layout, typography, component alignment,
+etc. 
+
+\begin{description}
+  \item[Front End Component Review:] We will engage a developer to look through
+  the code and interface and ensure components are implemented as they are
+  defined in the SRS.
+  \item[Accessibility Check:] A developer will review if accessibility tools
+  exist or not within the application. Important tools are screen readers and
```

Comment:
mention aria-label for screen reader

Reply:
also do we need to specify who the developer is? just wondering how specific we should be

Reply:
No, just need that _a_ developer is in charge of it